monitor/widgets/src/main/java/org/datacleaner/monitor/scheduling/widgets/SchedulePanel.java

```diff
@@ -70,19 +77,25 @@
     Button executeButton;
 
     @UiField
+    InlineLabel latestExecutionLabel;
+    
+    @UiField
     FlowPanel alertsPanel;
+    
```

Comment:
When I reformat the code in my IDE this line is removed (because there are now two consecutive empty lines.)